<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="side_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="patient_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Patients</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/patientdownload .png</normaloff>../Medico_modern/patientdownload .png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dash_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
border-bottom-color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dashboard.png</normaloff>../Medico_modern/dashboard.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="appointment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Appointments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/appointmentdownload (1).png</normaloff>../Medico_modern/appointmentdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="dentist_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Dentists</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/dentistimages.png</normaloff>../Medico_modern/dentistimages.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="payment_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Payments</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="expense_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>Expense</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Medico_modern/billingdownload (1).png</normaloff>../Medico_modern/billingdownload (1).png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="expense_desc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="expense_desc_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="expense_remark">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Remarks</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="expense_remark_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="expense_amount">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="expense_amount_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_expense_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 255);

border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
   </property>
   <property name="text">
    <string>Add Expense</string>
   </property>
  </widget>
  <widget class="QLabel" name="select_date">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Select a Date To See Expenses:</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
